server/plugins/connect-plugin-api/src/main/java/com/oceanbase/odc/plugin/connect/model/diagnose/PlanGraph.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.plugin.connect.model.diagnose;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.oceanbase.odc.common.unit.BinarySizeUnit;
+import com.oceanbase.odc.common.util.StringUtils;
+import com.oceanbase.odc.core.shared.model.QueryStatus;
+
+import lombok.Data;
+
+/**
+ * @author liuyizhuo.lyz
+ * @date 2024/4/11
+ */
+@Data
+public class PlanGraph {
```

Comment:
some fields are not easy to read

1. duration miss unit
2. overview contains Object as value of map
3. overview/statistics/topNodes，what is the key and value

often, map structure is a bad choice, as the map cannot explain itself.